node/messages/src/quorum.rs

```diff
@@ -0,0 +1,40 @@
+// Copyright (C) 2019-2023 Aleo Systems Inc.
+// This file is part of the snarkOS library.
+
+// The snarkOS library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// The snarkOS library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with the snarkOS library. If not, see <https://www.gnu.org/licenses/>.
+
+use super::*;
+use fastcrypto::bls12381::min_sig::{BLS12381PublicKey, BLS12381Signature};
+
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct Quorum {
```

Comment:
I think I'd call it a `ConsensusId`, `AuthorityId` or `BftId` in order to not suggest that this message on its own implies that there is a quorum or that it is only applicable until one is reached

Reply:
If we send a nonce to be signed instead, it should probably be a challenge-response nomenclature?

Reply:
could be :thinking: